manifests/a/Atlassian/Sourcetree/3.4.7/Atlassian.Sourcetree.installer.yaml

```diff
@@ -19,6 +19,14 @@ Installers:
     Custom: ACCEPTEULA=1
   UpgradeBehavior: install
   ProductCode: '{C2F82CF6-50FA-41FE-8000-1361A76223FA}'
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://product-downloads.atlassian.com/software/sourcetree/windows/ga/SourceTreeSetup-3.4.7.exe
+  InstallerSha256: 115F2EA206CDC1142865BA8FB11D3A8D6D20ACE9B82A5FE5CCCE8806C996B7C5
+  UpgradeBehavior: install
+  ProductCode: '{C2F82CF6-50FA-41FE-8000-1361A76223FA}'

```

Comment:
```suggestion
  ProductCode: 'SourceTree'
```

Reply:
![image](https://user-images.githubusercontent.com/11600822/146388863-3dcf0955-e603-4a3c-a294-0ef7532627e5.png)

Reply:
Good catch, thank you!